src/index.css

```diff
@@ -1,13 +1,38 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400&display=swap');
```

Comment:
- [ ] In the mobile view, the description form field is not consistent with the others:

![image](https://github.com/coderacheal/skill-habour-fe/assets/57421823/350747c0-9ca6-49d2-82a9-74b27e6b9ca3)

Kindly, add some styling so it's the same as the others.

That way, the form will have more consistency and the mobile UX will improve. 👍

Reply:
Sure we will fix it